<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font-family: Ravie;
font-size:18px;
color: rgb(185, 153, 25);
width: 20px;
</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="scoringLabel">
     <property name="styleSheet">
      <string notr="true">font-family: Ravie;
font-size:18px;
color: rgb(185, 153, 25);
width: 20px;
</string>
     </property>
     <property name="text">
      <string>Score: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>564</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="musicButton">
     <property name="styleSheet">
      <string notr="true">color: rgb(217, 217, 0);
background-color: rgb(39, 144, 37);
font-size: 18px;
font-family:Andalus;
height: 10px;
font-weight: bold;
</string>
     </property>
     <property name="text">
      <string>Music</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="minimumSize">
      <size>
       <width>565</width>
       <height>23</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(217, 217, 0);
background-color: rgb(39, 144, 37);
font-size: 18px;
font-family:Andalus;
height: 10px;
font-weight: bold;
</string>
     </property>
     <property name="text">
      <string>Pause/Resume</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
